### ISSUE TITLE
All Items on Cyber Sleuth removes all Equipment, Farm Goods and Medals

### ISSUE THREAD

**Issue by mc6415** [mineminemine/EdizonForDigimonCyberSleuthCE#2]
Using the 95x All Items on Cyber Sleuth removes everything outside of Usable Items and Key Items. Works fine on Hackers Memory.

**Comment by mc6415** [comment]
Well bizarrely after using an item and de-digivolving something now they're there.... colour me confused

**Comment by mineminemine** [comment]
Yeah the script for the x95 All Items is pretty janky which is why I was pretty reluctant to add the feature. Glad it somehow got fixed(?) for you though.
